scripts/publish.js

```diff
@@ -84,113 +85,168 @@ function cleanDistribution() {
 
     // (over-)write destination package.json file
-    const json = JSON.stringify(distPackageJson, null, 2);
+    const json = JSON.stringify(packageJson, null, 2);
     console.log(`Writing new package.json (to: ${distPackageJsonPath}):\n\n${json}`);
-    return fs.writeJson(distPackageJsonPath, distPackageJson, { spaces: 2 });
+    return fs.writeJson(distPackageJsonPath, packageJson, { spaces: 2 });
   });
 }
 
-function copyReadme() {
+function copyReadme(distTarget) {
   console.log('Copying README.md file...');
   return fs.copy('readme.md', `${distTarget}/readme.md`);
 }
 
-function forwardCoreScssFiles() {
-  console.log('Forwarding core SCSS files...');
-
+function createScssCoreForwardFiles(coreLibSrcDir, sccsCoreForwardFilePaths) {
   const sourceRootDir = `${coreLibDir}/scss`;
   const sharedRootDir = libsRootDir;
 
-  return new Promise((resolve) => {
-    pathsToForwardCoreScssFilesTo.forEach((targetRootDir) => {
-      forwardScssFiles({ sourceRootDir, targetRootDir, packageAlias, sharedRootDir });
-      resolve();
+  const copySccsCoreTargetFiles = () => {
+    console.log('Copying SCSS core target files...');
+
+    const onlyScssFiles = (input) => ['', '.scss'].includes(path.extname(input));
+    return fs.copy(`${coreLibSrcDir}/scss`, `${sourceRootDir}`, {
+      filter: onlyScssFiles,
     });
-  });
+  };
+
+  const generateSccsCoreForwardFiles = () => {
+    console.log('Creating SCSS core forwarding files...');
+    return new Promise((resolve) => {
+      sccsCoreForwardFilePaths.forEach((targetRootDir) => {
+        forwardScssFiles({ sourceRootDir, targetRootDir, packageAlias, sharedRootDir });
+        resolve();
+      });
+    });
+  };
+
+  const deleteSccsCoreTargetFiles = () => {
+    console.log('Deleting SCSS core target files...');
+    return fs.remove(`${sourceRootDir}`);
+  };
+
+  return copySccsCoreTargetFiles()
+    .then(generateSccsCoreForwardFiles)
+    .then(deleteSccsCoreTargetFiles);
 }
 
-function copyIcons() {
+function copyIcons(libSrcDir, distTarget) {
   console.log('Copying Icons...');
   const onlySvgFiles = (input) => ['', '.svg'].includes(path.extname(input));
-  return fs.copy(`${angularLibSrcDir}/icons/svg`, `${distTarget}/icons/svg`, {
+  return fs.copy(`${libSrcDir}/icons/svg`, `${distTarget}/icons/svg`, {
     filter: onlySvgFiles,
   });
 }
 
-function copyPolyfills() {
+function copyPolyfills(libSrcDir, distTarget) {
   console.log('Copying Polyfills...');
   const onlyLoadersAndMinified = (input) =>
     path.extname(input) === '' || input.endsWith('-loader.js') || input.endsWith('.min.js');
-  return fs.copy(`${angularLibSrcDir}/polyfills`, `${distTarget}/polyfills`, {
+  return fs.copy(`${libSrcDir}/polyfills`, `${distTarget}/polyfills`, {
     filter: onlyLoadersAndMinified,
   });
 }
 
-function createTarballPackage() {
+function copyCoreStencilBuildFiles(stencilBuildDir, distTarget) {
+  console.log('Copying core stencil build files...');
```

Comment:
Consider renaming this, as there is no stencil-specifics at this time. 
It should all be compiled to something the browser understands, so maybe copyCoreDistributionFiles is a better name?

```suggestion
function copyCoreDistributionFiles(stencilBuildDir, distTarget) {
  console.log('Copying core distribution files...');
```